projects/telemetry/src/gps.c

```diff
@@ -46,38 +34,13 @@ StatusCode gps_init(GpsSettings *settings, GpsStorage *storage) {
   s_settings = settings;
   s_storage = storage;
 
-  GpioSettings telemetry_settings_gpio_general = {
-    .direction = GPIO_DIR_OUT,  // The pin needs to output.
-    .state = GPIO_STATE_LOW,    // Start in the "off" state.
-    .alt_function = GPIO_ALTFN_1,
-  };
-
   // Initializes UART
   uart_init(s_settings->port, s_settings->uart_settings, &s_settings->uart_storage);
```

Comment:
Let's actually leave `uart_init` out, and instead do it in the main method. We will expect users to initialize uart for the GPS driver and they will only pass us the `UartPort`. That should simplify the design